python/fichas/dicionarios.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+"""
+# Ficha x: Questões sobre Dicionários
+
+Assim como as listas, os Dicionários são usados para armazenar dados em uma única variável.
+No entanto, a sua diferença encontra-se principalmente nos índices e nos seus métodos:
+Uma lista possui int's como índice, enquanto que um dicionário possui quase qualquer tipo de dados.
+Os dicionários possuem o que chamamos de chaves (keys) e valores (values) para fazer a associação.
+"""
+
+
+def traducoes(d: dict) -> list:
+    """
+    Apresente uma definição da função traducoes(d: dict) -> list: que dado um Dicionário
```

Comment:
```suggestion
    Dado um Dicionário
```
You should omit the first part since the build script would automatically do it. Which means we can just read as it is.